resource_types/core/401-paygcredit/paygcredit.oas.yaml

```diff
@@ -0,0 +1,254 @@
+        (https://www.oneiota.org/revisions/5310).
+
+        Any device may be 'independent'.
+        Controller-capable devices may be 'leading'.
+        Accessory-capable devices may be 'following'.
+        
+        A "Following" device has PAYG credit managed by another device which is
+        "Leading".
+
+        "Relaying" is not implemented in this release. A "Relaying" device
+        has PAYG credit managed by another device which is "Leading", but
+        a third device follows the "Relaying" device ('chaining' the credit
+        from device A to device C via device B, even though devices A and C
+        are not directly connected).
+
+        The default interface is 'oic.if.rw'. To retrieve the baseline OCF
+        information, it is necessary to query with 'if=oic.if.baseline'.
+
+        Application security constraints -
+        When used with Nexus Channel, the device POSTing must be a controller
+        with an active Nexus Channel Link to this device.
+
+        See also -
+        * angaza.com.nexus.channel.link_handshake - Information on Nexus channel link handshakes
+          
+      parameters:
+        # - $ref: '#/parameters/interface'
+      responses:
+        200:
+          description: "PAYG Credit specific information about this device."
+          schema:
+            $ref: '#/definitions/FullPAYGState'
+          x-example:
+            description: "x-example included only for automatic code generation."
+            rt:
+              - angaza.com.nexus.channel.core.paygcredit
+            if:
+              - oic.if.rw
+              - oic.if.baseline
+            re: 603755
+            un: 1
+            mo: 0
+
+    post:
+      summary: Set remaining PAYG credit of the device
+      description: |
+        Sets the remaining PAYG credit of the device. If the endpoint
+        determines that the requesting device is not authorized to control
+        this device, an error response will be returned.
+      parameters:
+        - in: body
+          name: Set Remaining PAYG Credit
+          description: Amount of PAYG credit to set on this device. A value of '4294967295' will "Unlock" the device.
+          schema:
+            $ref: "#/definitions/FullPAYGState"
+
+      responses:
+        200:
+          description: Successfully updated credit (Response payload contains new credit)
+          schema:
+              properties:
+                re:
+                  $ref: '#/definitions/FullPAYGState/properties/re'
+        400:
+          description: Credit amount/value not supported.
+        401:
+          description: Client is not authorized to perform this action (client may lack a Nexus Channel Link to this device)
+        405:
+          description: This device is not currently capable of remotely updating credit (may be in 'independent' mode)
+          schema:
+              properties:
+                mo:
+                  $ref: '#/definitions/FullPAYGState/properties/mo'
```

Comment:
I think it does not now actually do this (send `mo` with the error response)?

Reply:
Will delete, to add as a later addition if needed.

Reply:
Updated to remove this